### DISCUSSION TITLE
Current User email in REST call

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3879] [Ideas]
Since the REST Api call is server-side and the `Access level` checks for authenticated request, is there a was to end details about the authenticated user as part of the call.

`{{ Current User.email }}` isn't accepted as part of the `POST` request body. This would be an awesome feature to track the originating user from the REST endpoint.

The `authenticated_user_email` parameter should not be sent from the client-side to avoid impersonation

**Comment by mjashanks** [comment]
Agreed - logged this here: https://github.com/Budibase/budibase/issues/3884
